server/src/test/java/io/spine/server/stand/StandPostShould.java

```diff
@@ -163,103 +107,149 @@ private static void checkUpdatesDelivery(boolean isConcurrent,
         verify(stand, times(dispatchActions.length)).update(any(EntityStateEnvelope.class));
     }
 
+    @SuppressWarnings("CheckReturnValue") // can ignore the dispatch() result
     private static BoundedContextAction aggregateRepositoryDispatch() {
-        return new BoundedContextAction() {
-            @Override
-            public void perform(BoundedContext context) {
-                // Init repository
-                final AggregateRepository<?, ?> repository = Given.aggregateRepo();
-
-                repository.initStorage(storageFactory(context.isMultitenant()));
-
-                try {
-                    // Mock aggregate and mock stand are not able to handle events
-                    // returned after command handling.
-                    // This causes IllegalStateException to be thrown.
-                    // Note that this is not the end of a test case,
-                    // so we can't just "expect=IllegalStateException".
-                    final CommandEnvelope cmd = CommandEnvelope.of(Given.validCommand());
-                    repository.dispatch(cmd);
-                } catch (IllegalStateException e) {
-                    // Handle null event dispatching after the command is handled.
-
-                    // Check if this error is caused by returning nuu or empty list after
-                    // command processing.
-                    // Proceed crash if it's not.
-                    if (!e.getMessage()
-                          .contains("No record found for command ID: EMPTY")) {
-                        throw e;
-                    }
+        return context -> {
```

Comment:
It would be better to inline this lambda body with the method body and use the method reference.